testhelper/certificatetest.go

```diff
@@ -111,6 +131,21 @@ func getRSACertTuple(cn string, issuer *RSACertTuple) RSACertTuple {
 	return GetRSACertTupleWithPK(pk, cn, issuer)
 }
 
+func getRevokableRSACertTuple(cn string, issuer *RSACertTuple) RSACertTuple {
+	template := getCertTemplate(issuer == nil, true, cn)
+	template.OCSPServer = []string{"http://localhost:8080/ocsp"}
```

Comment:
It doesn't actually use port 8080 since the request should be intercepted by the mock client. However, I'm considering changing it to `http://example.com/chain_ocsp/%d` (tracked [here](https://github.com/notaryproject/notation-core-go/pull/134#discussion_r1150779037))